default.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "group:recommended",
+    ":separateMultipleMajorReleases",
+    ":pinDevDependencies",
+    "github>smg-automotive/renovate-config:groupDevDependencies",
+    "github>smg-automotive/renovate-config:nodeUpdate",
+    "github>smg-automotive/renovate-config:dockerOS",
+    "github>smg-automotive/renovate-config:internalPackageUpdate"
+  ],
+  "reviewers": [
+    "team:frontend"
+  ],
+  "node": {
+    "supportPolicy": [
+      "lts_latest"
```

Comment:
I remember this not working as expectected and if my memory does not fail me, this was not included anymore in the latest docs. can you please double check if there's a new way for specifying lts preference?

Reply:
looks like it's not possible to do it at all https://github.com/renovatebot/renovate/discussions/14909

we define it as `FROM node:lts-alpine`

Reply:
works for the docker containers and locally we always relied on manual switches